### DISCUSSION TITLE
copycat site on HN front page

### DISCUSSION THREAD

**Discussion by navels** [tteck/Proxmox#3545] [General]
Link: : https://news.ycombinator.com/item?id=41282981

Not sure what the point of this is . . .

**Comment by tteck** [comment]
Some may take advantage of the project's popularity to gain their 15 minutes of fame or potentially inject malicious code. As a result, third-party front-ends or cloned sites are likely to emerge. I take every precaution to ensure the security of this repository and have posted warnings, but I guess some will learn the hard way.
